rbi/yast/client.rbi

```diff
@@ -0,0 +1,9 @@
+# typed: strong
+
+module Yast
+  class Client
+    include I18n
```

Comment:
is this needed? Cannot it load just from rb file?

Reply:
Sorbet takes great care to be fast so I guess the major reason is to be able to read just the RBI files of your dependencies and not parse and analyze all their sources.